snykTask/task.json

```diff
@@ -173,7 +173,6 @@
       "groupName": "advanced"
     }
   ],
-  "minimumAgentVersion": "3.232.1",
```

Comment:
issue: removing this might be another breaking change (see [here](https://github.com/microsoft/azure-pipelines-tasks/blob/master/docs/migrateNode20.md#specify-minimumagentversion)):

suggestion: set the minimum version to the first version that supports node10:
`"minimumAgentVersion": "2.144.0"`

Reply:
I agree that it seems better practice to explicitly state the minimum agent version, however I believe this represents a breaking change for our consumers. So in this instance I would like to revert to the previous state where the agent version was undefined.